Source/options.h

```diff
@@ -92,6 +92,8 @@ struct GameplayOptions {
 	bool bAutoGoldPickup;
 	/** @brief Recover mana when talking to Adria. */
 	bool bAdriaRefillsMana;
+	/** @brief Automatically attempt to equip items when picking them up. */
+	char szAutoEquipItems[100];

```

Comment:
`std::string`?

Reply:
I tried initially, but  get/setIniValue() prefers char []. And most other stings in options structures use char [], not std::string.